molecule/workstation/playbook.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Bootstrap user env.
+  hosts: all
+  remote_user: root
+  tasks:
+    - name: Set passwordless sudo.
+      lineinfile:
+        dest: /etc/sudoers
+        regexp: '^%sudo'
+        line: '%sudo ALL=(ALL) NOPASSWD:ALL'
+        validate: '/usr/sbin/visudo -cf %s'
+
+- name: Configure kernel build.
+  hosts: all
+  pre_tasks:
+    # You can set these values via env vars:
+    #
+    #   * GRSECURITY_USERNAME
+    #   * GRSECURITY_PASSWORD
+    #
+    # The role will then automatically pick them up and these assert
+    # statements will pass.
+    - name: Check for grsecurity login credentials.
+      assert:
+        that:
+          - grsecurity_build_download_username != ''
+          - grsecurity_build_download_password != ''
+  remote_user: vagrant
+  environment:
+    USER: vagrant
+  roles:
+    - role: ansible-role-grsecurity-build
+      grsecurity_build_patch_type: stable3
+      grsecurity_build_strategy: manual
```

Comment:
do we want this to be manual build strategy? I think its fine, its easy to over-ride with `molecule converge -- -e=` pragma -  I'm just making sure this is what you intended long-term.

Reply:
We could set `grsecurity_build_custom_config: 'config-workstation-4.14'` if overriding is tedious, but the manual strategy is best human usage.